tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -72,13 +72,27 @@ def parameters(cls):
         return parameters
 
     def configure(self):
+        def is_detection_out(config):
+            if config.get('detection_out'):
+                return True
+
+            return False
+
+        def is_box_outputs(config, box_outputs):
+            for elem in box_outputs:
+                if not config.get(elem):
+                    return False
+
+            return True
+
         box_outputs = ['classes_out', 'scores_out', 'boxes_out']
-        detection_out = 'detection_out'
-        if contains_all(self.launcher_config, [*box_outputs, detection_out]):
+        if is_detection_out(self.launcher_config) and is_box_outputs(self.launcher_config, box_outputs):
             raise ConfigError('only detection output or [{}] should be provided'.format(', '.join(box_outputs)))
-        self.detection_out = self.get_value_from_config(detection_out)
+
+        self.detection_out = self.get_value_from_config('detection_out')
```

Comment:
at this moment, you already know which output should be, please use check function results for configure outputs

Reply:
Done